docs/ssss.md

```diff
@@ -1,3 +1,94 @@
 # Simperby Super Simple Summary
 
-TODO
+## DAO
+
+A DAO (Decentralized Autonomous Organization) is an organization that runs on a
+blockchain, governed by rules set in a contract, and operates autonomously
+without any government backing.
+
+For more information on DAOs, refer to [the introduction to DAO
+(WIP)](./dao.md).
+
+## How to Implement a DAO
+
+The most common way to implement a DAO is by **using a mainnet that supports
+smart contracts**. You can code your governance rules in a contract and deploy
+it on the mainnet, which will interact with the mainnet's ecosystem. However,
+this approach **has some disadvantages**:
+
+1. It requires payment of a gas fee.
+2. It is dependent on the chosen mainnet's performance, security, reliability,
+   usability, political risks, etc.
+3. It is limited to the ecosystem of the chosen mainnet and cannot easily
+   interact with other mainnets.
+
+Simperby solves these problems by providing a standalone blockchain for a DAO.
+
+For more details, refer to [the comparisons (WIP)](./comparison.md)
+
+## Simperby
+
+Simperby is a blockchain engine that builds a standalone blockchain for a DAO.
+
+- Trustless
+- Distributed
+- Fault Tolerant
+- Lightweight
+- Highly Interoperable
+- Native Support for Multichain Treasuries
+
+For details on the core protocol, refer to the [protocol
+overview](./protocol_overview.md).
+
+### A Byzantine Fault Tolerant System
+
+- **Builds an independent layer-1 mainnet** like [Cosmos
+  SDK](https://v1.cosmos.network/sdk) or [Substrate](https://substrate.io/).
+- **One mainnet for one DAO**.
+- Has own **BFT consensus** called [Vetomint](./vetomint.md) optimized for
+  sporadic node operations.
+
+### Governance
+
+- A peer-to-peer voting mechanism (governance) is used to make decisions on the
+organization, which is then finalized by the consensus.
+- No tokens, no staking, no mining, no gas fee; Instead, **permissioned and
```

Comment:
Both `no tokens' and 'explicitly nominated nodes' mean that this system is not a PoS.
Consensus voting power is granted explicitly (not from owning a native token).

Reply:
I got it. How about adding some description to explain why this is beneficial? (probably efficiency?)

Reply:
This is not a strategical decision; rather it's an inevitable consequence of sporadic node operation because this system can't handle general transactions such as token transfers. A token system can't exist because of this architecture, so there is no way to establish PoS (which requires tokens).

Reply:
The benefits here is the simplicity of the protocol. Have and idea to effectively deliver this notion?

Reply:
I'm not sure what should be the best expression but I understand your concern. Let's keep it as is for now.